sqs/src/test/scala/docs/scaladsl/SqsPublishSpec.scala

```diff
@@ -43,18 +44,20 @@ class SqsPublishSpec extends AnyFlatSpec with Matchers with DefaultTestContext w
         .asScala
         .head
 
-    def receiveMessages(maxNumberOfMessages: Int): Seq[Message] = {
+    def receiveMessages(maxNumberOfMessages: Int, waitTimeSeconds: Int = 0): Seq[Message] = {
       // see https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sqs/model/ReceiveMessageRequest.html
       require(maxNumberOfMessages > 0 && maxNumberOfMessages <= 10, "maxNumberOfMessages must be in 1 to 10")
 
+      val timeoutSeconds = messageRecieveMinimumTimeout + waitTimeSeconds
```

Comment:
There's a typo in message**Recieve**MinimumTimeout.

Reply:
Updated

Also squashed commits